packages/ETLoad.package/ETLDirectoryTestCase.class/instance/doesNotUnderstand..st

```diff
@@ -0,0 +1,9 @@
+private
+doesNotUnderstand: aMessage
+	"intercept test messages and forward them to #testFile with
+	the file name extracted from the message selector."
+
+	(aMessage selector startsWith: 'test') ifTrue: [
+		^ self testFile: (aMessage selector allButFirst: 4). ].
+	
+	^ super doesNotUnderstand: aMessage.
```

Comment:
I think we should either change the names of our `*.ims` files to be more descriptive, or allow subclasses of `ETLDirectoryTestCase` to specify a prefix. A test named `test42` is not really descriptive, and especially because we use quite long test names in all other places. I would suggest something like `testImageSegmentContaining42` or `testImageSegment42`.